mkosi

```diff
@@ -1496,6 +1492,10 @@ SigLevel    = Required DatabaseOptional
     with open(os.path.join(workspace, 'root', 'etc/locale.conf'), 'w') as f:
         f.write('LANG=en_US.UTF-8\n')
 
+    # At this point, no process should be left running, kill then
+    run(["fuser", "-c", root, "--kill"])
```

Comment:
on line 1480 you are killing gpg explicitly, and your comment says dirmngr stays alive.Why is that not a bug in gpg?

Reply:
Are they the same thing? I do see a "bye" as a result from this command. I thought it was another process that was left running.

Reply:
Seems to be a bug in gpg. Killing stuff like that is too ugly.

Reply:
@falconindy any pointer on the pacstrap/pacman side where the fix for this problem would be? For context, it's agout dirmngr left running after those command return.